sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -111,8 +98,8 @@ export function formatWildcards(
   }
 
   return {
-    key,
-    label,
+    key: listConfigOptions.keyFilter,
```

Comment:
I'm not sure where this return value goes, but if we're turning a keyFilter directly into a key, we'd want to make sure it was correctly escaped.  Would it cause a lot of disruption to have the contract for this method return a keyFilter instead?

Reply:
I think we're good based on your other comment but just in case - this is going from the 'customer facing' contract (which has .keyFilter and .labelFilter) and going to to the internal service API representation (which still takes '.key' and '.label')